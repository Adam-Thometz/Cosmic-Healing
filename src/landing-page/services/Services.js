import React from 'react';

import './Services.css';

import SectionHeader from '../../_components/section-header/SectionHeader';
import SmallCaption from '../../_components/small-caption/SmallCaption';
import ServiceCard from './service-card/ServiceCard';

import options from './serviceOptions';

const Services = () => {
  return (
    <section className='Services'>
      <div className='Services-header'>
        <SmallCaption text="DO WHAT IS BEST FOR YOU" />
        <SectionHeader text="Choose your ideal environment" />
      </div>
      <div className='Services-options'>
        {options.map(o => (
          <ServiceCard 
            service={o.service}
            description={o.description}
            link={o.link}
          />
        ))}
      </div>
    </section>
  );
};

export default Services;