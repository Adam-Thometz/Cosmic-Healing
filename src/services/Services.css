.Services {
  background-image: 
    url(../_media/backgrounds/orange-lines.png),
    url(../_media/backgrounds/orange.png);
  background-size: 90% 100%, 100% 100%;
  background-repeat: no-repeat;
  height: 600px;
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  margin: 50px 0;
}

.Services-options {
  display: flex;
  width: 80%;
  margin: 0 auto;
}