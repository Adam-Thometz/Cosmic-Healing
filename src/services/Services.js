import React from 'react';

import './Services.css';

import SectionHeader from '../_components/section-header/SectionHeader';
import SmallCaption from '../_components/small-caption/SmallCaption';
import ServiceDescription from './service-description/ServiceDescription';
import FAQ from './faq/FAQ';

import serviceDescriptions from './serviceDescriptions';

const Services = () => {
  const services = serviceDescriptions.map(({
    name,
    description,
    image
  }, i) => {
    return <ServiceDescription
      style={{ "--i": i+2 }}
      name={name}
      description={description}
      image={image}
      i={i}
    />
  });
  
  return (
    <main className='Services'>
      <section style={{ "--i": 1 }}>
        <SmallCaption text='DO WHAT IS BEST FOR YOU' />
        <SectionHeader text="Choose your ideal environment" />
      </section>
      {services}
      <FAQ style={{ "--i": 4 }} />
    </main>
  );
};

export default Services;