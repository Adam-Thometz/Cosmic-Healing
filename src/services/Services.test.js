import React from "react";
import { render } from "@testing-library/react";
import Services from './Services';

describe('Services component', () => {
  it('renders without crashing', () => {
    render(<Services />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<Services />);
    expect(asFragment()).toMatchSnapshot();
  });
});