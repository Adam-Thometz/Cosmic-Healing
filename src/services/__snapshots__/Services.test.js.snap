// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services component matches the snapshot 1`] = `
<DocumentFragment>
  <main
    class="Services"
  >
    <small
      class="SmallCaption"
      style="color: black;"
    >
      DO WHAT IS BEST FOR YOU
    </small>
    <h2
      class="SectionHeader"
    >
      Choose your ideal environment
    </h2>
    <section
      class="ServiceDescription"
    >
      <img
        alt=""
        src="astronaut-single.png"
      />
      <aside
        class="ServiceDescription-description"
      >
        <h3
          class="SectionHeader"
        >
          Individual sessions
        </h3>
        <article>
          Work one-on-one with a trained professional to uncover parts of yourself and heal from trauma. Sessions are confidential and personalized to your unique goals. This approach is recommended if you prefer a more intimate therapist-client relationship.
        </article>
        <a
          class="Link primary"
        >
          Book a free session
        </a>
      </aside>
    </section>
    <section
      class="ServiceDescription reverse"
    >
      <img
        alt=""
        src="astronaut-group.png"
      />
      <aside
        class="ServiceDescription-description"
      >
        <h3
          class="SectionHeader"
        >
          Group sessions
        </h3>
        <article>
          A group setting can help you gain clarity and perspective on your own life as you begin to cultivate relations with others. Members go through a vetting process and are placed based on their unique experience and goals as it relates to the groupâ€™s overall dynamnic.
        </article>
        <a
          class="Link primary"
        >
          Book a free session
        </a>
      </aside>
    </section>
    <section
      class="FAQ"
      id="faq"
    >
      <small
        class="SmallCaption"
        style="color: black;"
      >
        FAQ
      </small>
      <h2
        class="SectionHeader"
      >
        Frequently Asked Questions
      </h2>
      <div
        class="Question"
        id="0"
      >
        <div
          class="Question-question-wrapper"
        >
          <h3
            class="SectionHeader"
          >
            How much does a session cost?
          </h3>
          <img
            alt="Pick to expand question"
            src="expanded.png"
          />
        </div>
        <p
          class="Question-answer"
        >
          Interaction design is the practice of designing interactive digital products, environments, systems, and services.
        </p>
      </div>
      <div
        class="Question"
        id="1"
      >
        <div
          class="Question-question-wrapper"
        >
          <h3
            class="SectionHeader"
          >
            Where are sessions held?
          </h3>
          <img
            alt="Pick to expand question"
            src="expanded.png"
          />
        </div>
        <p
          class="Question-answer"
        >
          Interaction design is the practice of designing interactive digital products, environments, systems, and services.
        </p>
      </div>
      <div
        class="Question"
        id="2"
      >
        <div
          class="Question-question-wrapper"
        >
          <h3
            class="SectionHeader"
          >
            What is your deposit policy?
          </h3>
          <img
            alt="Pick to expand question"
            src="expanded.png"
          />
        </div>
        <p
          class="Question-answer"
        >
          Interaction design is the practice of designing interactive digital products, environments, systems, and services.
        </p>
      </div>
      <div
        class="Question"
        id="3"
      >
        <div
          class="Question-question-wrapper"
        >
          <h3
            class="SectionHeader"
          >
            What insurance do you accept?
          </h3>
          <img
            alt="Pick to expand question"
            src="expanded.png"
          />
        </div>
        <p
          class="Question-answer"
        >
          Interaction design is the practice of designing interactive digital products, environments, systems, and services.
        </p>
      </div>
    </section>
  </main>
</DocumentFragment>
`;
